src/Http/Http.Extensions/src/ContextExtensions.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Http.Features;
+
+namespace Microsoft.AspNetCore.Http
+{
+    public static class ContextExtensions
```

Comment:
Extension method classes are supposed to be named [ObjectBeingExtended]+[FeatureAreaBeingAdded] to avoid conflicts with other extension methods on the same type. Especially when they all use the same namespace.
```suggestion
    public static class HttpContextServerVariableExtensions
```